rdrf/rdrf/initial_data/__init__.py

```diff
@@ -1,5 +1,5 @@
 from . import (base, groups, users, sample_registry, sample_laboratories,
-               genes, genes_smaller_sample, DEV, iprestrict_permissive, reference_data)
+               genes, genes_smaller_sample, DEV, iprestrict_permissive, reference_data, patient_stage)
```

Comment:
Needed? `patient_stage` is part of `reference_data` now.

Reply:
Updated.